ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -140,15 +165,44 @@ export default {
           fr: 'Gauche, puis droite',
           ja: '左 => 右',
           cn: '左 => 右',
-          ko: '왼쪽 => 오른쪽',
+          ko: '왼쪽, 그리고 오른쪽',
```

Comment:
https://github.com/quisquous/cactbot/blob/95d2ea004c246830d19a2307389248d907485abd/ui/raidboss/popup-text.js#L945

The code above has already replaced the arrow ("=>") into "geuligo"(sorry I can't type korean) when in TTS.
You don't need to do these fixes here and below.

Reply:
I don't know but, modified as your request.